### PR TITLE
ADLPlug: v1.0.0-beta.5 -> v1.0.0

### PR DESCRIPTION
**Pull request by ghost**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by veprbl** [comment]
@GrahamcOfBorg build adlplug opnplug

File: pkgs/applications/audio/adlplug/default.nix [hunk 1]
```diff
@@ -5,13 +5,13 @@
 
 stdenv.mkDerivation rec {
   name = "${pname}-${version}";
```
Comment (ryantm) [inline pkgs/applications/audio/adlplug/default.nix]:
```suggestion
  inherit pname;
```

File: pkgs/applications/audio/adlplug/default.nix [hunk 2]
```diff
@@ -5,13 +5,13 @@
...
-  version = "v1.0.0-beta.5";
+  version = "v1.0.0";
```
Comment (ryantm) [inline pkgs/applications/audio/adlplug/default.nix]:
```suggestion
  version = "1.0.0";
```
It's not typical to include a v prefix in the version string.

File: pkgs/applications/audio/adlplug/default.nix [hunk 3]
```diff
@@ -5,13 +5,13 @@
...
 
   src = fetchFromGitHub {
     owner = "jpcima";
     repo = "ADLplug";
     rev = version;
```
Comment (ryantm) [inline pkgs/applications/audio/adlplug/default.nix]:
```suggestion
    rev = "v${version}";
```

**Review by ryantm — CHANGES_REQUESTED** [review]

**Comment by gnidorah** [comment]
@ryantm Fixed, thanks!

**Comment by gnidorah** [comment]
@veprbl Thank you!

**Pull request merged**